infrastructure/network/rpcclient/log.go

```diff
@@ -0,0 +1,9 @@
+package rpcclient
+
+import (
+	"github.com/kaspanet/kaspad/infrastructure/logger"
+	"github.com/kaspanet/kaspad/util/panics"
+)
+
+var log, _ = logger.Get(logger.SubsystemTags.RPCC)
```

Comment:
You need to define UseLogger function, otherwise you'll be able to use the RPCClient only with kaspad logger, which is not very useful